docs/style-guide.rst

```diff
@@ -809,7 +809,21 @@ possible permutations for function declarations.
 Mappings
 ========
 
-TODO
+Surround the type of a ``mapping`` by parentheses. In variable declarations, separate the keyword ``mapping`` from its type by a single space. Do not separate any nested ``mapping`` keyword from its type by whitespace.
```

Comment:
Is it really standard practice to change the spacing for nested mappings?

Reply:
I don't know, I am merely adding the result of the conversation from https://github.com/ethereum/solidity/issues/2917 if there's something different let me know.